src/docs/product/data-management-settings/event-grouping/merging-issues/index.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: "Merging Issues"
+sidebar_order: 40
+description: "Learn about merging similar issues that aren't automatically grouped."
+---
+
+If you have similar-looking issues that have not been grouped automatically and you want to reduce noise, you can do so by merging them.
+
+This is for merging similar issues and will not auto-merge the next occurrence of this issue coming in.
+<!-- what exactly what does the statement above mean? Do we mean, this won't happen if the next event doesn't have the same stack trace? See note at end.-->
```

Comment:
@untitaker @jjbayer I'm not sure what we mean here as it pertains to what's on line 23. Can you clarify?

Reply:
I don't know where this content comes from but essentially it means that this is a one-off operation, and we don't try to infer any "rules" from a user merging two issues together. I would omit the sentence entirely because I can't think of a better way to explain and I don't think it's relevant at this stage. Perhaps we can point to fingerprinting rules at the bottom of the page, as a way to tweak grouping more aggressively if issue merging doesn't cut it. Replace alert at the bottom with:

>Future events will be added to the merged issue by the same criteria as they would've been added to the duplicate issues. We don't try to infer any general rules from how you merge issues. If you see that Sentry creates more issues than you can merge, or that Sentry creates too few issues, check out Fingeprinting Rules and Stack Trace Rules.